.drone.jsonnet

```diff
@@ -34,12 +42,26 @@ local stable_distros = [
   'amazon-2',
   'centos-6',
   'centos-7',
+  'centos-8',
   'debian-8',
   'debian-9',
+  'debian-10',
```

Comment:
debian 10 doesn't support python2, which I think would be triggered here. We could try git installs if you want...